core/templates/dev/head/pages/exploration_player/PlayerTranscriptService.js

```diff
@@ -67,19 +60,9 @@ oppia.factory('PlayerTranscriptService', [
         // TODO(aks681): Once worked examples are introduced, modify the below
         // line to take into account the number of worked examples displayed.
         var previousCard = angular.copy(transcript[transcript.length - 2]);
-        previousCard.setLeadsToConceptCard(false);
+        previousCard.markAsNotCompleted();
```

Comment:
Hm, in general I think we should pull some of this stuff out into a ConceptCardEngineService, and have ExplorationEngineService cache a copy of the old card so that it can re-send it when it's time to use it again. But we can do that in a future PR.

Reply:
Ok, will file issue for it.

Reply:
Done, created [issue](https://github.com/oppia/oppia/issues/5566)